benchkit/commandwrappers/trace-cmd.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 Huawei Technologies Co., Ltd. All rights reserved.
+# SPDX-License-Identifier: MIT
+"""
+Command wrapper for the `strace` utility that allows to trace all system calls used in the wrapped
+command and output them into a file.
+"""
+
+import os
+from typing import List, Optional
+
+from benchkit.commandwrappers import CommandWrapper
+from benchkit.dependencies.packages import PackageDependency
+from benchkit.utils.types import PathType
+
+
+class TraceCmdWrap(CommandWrapper):
```

Comment:
perhaps you can re-use some components of the other tracecmd module by importing them. But for now it is sufficiently simple as it is.